ansible/playbook.yaml

```diff
@@ -151,24 +167,39 @@
         owner: root
         group: root
         mode: '0644'
-    - name: systemd template
+    - name: web systemd template
       ansible.builtin.template:
         src: sfs.service.j2
         dest: /etc/systemd/system/sfs.service
         owner: root
         group: root
         mode: '0644'
+    - name: worker systemd template
+      ansible.builtin.template:
+        src: sfsworker.service.j2
+        dest: /etc/systemd/system/sfsworker.service
+        owner: root
+        group: root
+        mode: '0644'
     - name: systemd reload
       ansible.builtin.systemd:
         daemon_reload: true
-    - name: systemd enable
+    - name: web systemd enable
       ansible.builtin.systemd:
         name: sfs
         enabled: yes
-    - name: systemd start
+    - name: web systemd start
       ansible.builtin.systemd:
         name: sfs
         state: restarted
```

Comment:
I know this is off topic for the PR, but just mentioning it in passing...

One day can add ansible handlers to only restart when something has changed that impacts it https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_handlers.html

An example from karrot:
- this task calls "notify" after changing the env config https://github.com/karrot-dev/yuca/blob/26e36067c42290481a654bf42dc79646c3e249cc/roles/karrot-backend/tasks/main.yml#L372
- it causes this handler to run and restart https://github.com/karrot-dev/yuca/blob/26e36067c42290481a654bf42dc79646c3e249cc/roles/karrot-backend/handlers/main.yml#L3-L8

Reply:
I see, excellent, taking note!